client/src/locales/fr-FR.json

```diff
@@ -491,7 +492,8 @@
                 "allowed": "Activé",
                 "forbidden": "Désactivé",
                 "title": "Configuration",
-                "serverAddr": "Adresse du serveur"
+                "serverAddr": "Adresse du serveur",
+                "copyPath": "Copier"
```

Comment:
Displayed when the button has been clicked, so it should be `Copié`. Or `Copié !`